src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -278,6 +278,7 @@ CommandsRegistry.registerCommand({
 		} catch (e) {
 			onUnexpectedError(e);
 		}
+		return;
```

Comment:
?

Reply:
All these async functions reported `Not all code paths return a value` once I bumped the node version. I just added returns to squeltch the error. I've no idea why it only started being reported then, honestly.